src/overlays/actors/ovl_Bg_Po_Syokudai/z_bg_po_syokudai.c

```diff
@@ -1,15 +1,48 @@
+/*
+ * File: z_bg_po_syokudai.c
+ * Overlay: ovl_Bg_Po_Syokudai
+ * Description: Golden Torch Stand (Poe Sisters)
+ */
+
 #include "z_bg_po_syokudai.h"
 
 #define FLAGS 0x00000000
 
 #define THIS ((BgPoSyokudai*)thisx)
 
+#define POE_SISTER_PURPLE 0 // Meg
+#define POE_SISTER_RED 1    // Joelle
+#define POE_SISTER_BLUE 2   // Beth
+#define POE_SISTER_GREEN 3  // Amy
```

Comment:
This should probably be an enum, and maybe it should be specific to the type of torch rather than the type of poe sister.

Reply:
I've pushed a new version with the macros removed, s/poeSisterColor/flameColor/, and replaced the hard-coded colors with an enum. Please have a look and let me know.